x-pack/test_serverless/api_integration/test_suites/security/platform_security/authorization.ts

```diff
@@ -839,12 +839,12 @@ export default function ({ getService }: FtrProviderContext) {
                 "saved_object:search-session/delete",
                 "saved_object:search-session/bulk_delete",
                 "saved_object:search-session/share_to_space",
-                "ui:discover_v2/show",
```

Comment:
>[!CAUTION]
> The change with Discover V2 (https://github.com/elastic/kibana/pull/202863) was never backported to 8.x/8.18/8.17. Someone mistakenly resolved conflicts in this file while backporting their changes to 8.x, and since these tests don't seem to run in 8.x (I'll double-check with @elastic/appex-qa), CI didn't complain. I've fixed this and tested it locally:

```bash
node scripts/functional_tests_server.js --config x-pack/test_serverless/api_integration/test_suites/security/config.feature_flags.ts
---
node scripts/functional_test_runner.js --config x-pack/test_serverless/api_integration/test_suites/security/config.feature_flags.ts --grep "composite features"
```